js-core/homeworks/homework-1/src/main.js

```diff
@@ -0,0 +1,24 @@
+var userName = "Albert Einstain"; // string
+var isReady = false; //boolean
+var phoneNumber = 1111111; //number 
+var nullField = null; //null
+var emptyField = undefined; //undefined
+var parents = {
+		"father" : "unknown",
+		"mother" : " Elizabet Taylor"
+	}; // object 
+var friends = ['Boris', 'Stepan', 'Taras']; //array (object)
+
+var user = {
+	"isReady" : isReady,
```

Comment:
it's not necessary to use double quotes if you add a property to object.

Look good overall